examples/06-custom-schema/05-alert-block-full-ux/.bnexample.json

```diff
@@ -0,0 +1,10 @@
+{
+  "playground": true,
+  "docs": true,
+  "author": "matthewlipski",
+  "tags": ["Intermediate", "Blocks", "Custom Schemas", "Formatting Toolbar", "Suggestion Menus", "Slash Menu"],
```

Comment:
I personally don't like Intermediate and Advanced. It's just what it takes and it isn't about the person's skill level, it's about how easy we make it for them. It's a failing to have many things as "advanced"

Reply:
I definitely agree, however I don't think it's worth changing this for the examples at the moment since these tags are not used anywhere. We originally added them because we were floating the idea of using them to filter examples instead of using sidebar categories, but never got around to that.

Imo we should revisit this when working on the big docs update, since it's also relevant for information hierarchy.